jobbergate-api/jobbergate_api/apps/job_submissions/routers.py

```diff
@@ -348,6 +349,7 @@ async def job_submissions_agent_active(
     """
     identity_claims = IdentityClaims.from_token_payload(token_payload)
     if identity_claims.cluster_id is None:
+        logger.error("Access token  does not contain a cluster_id")
```

Comment:
```suggestion
        logger.error("Access token does not contain a cluster_id")
```

Is there a reason to have two spaces?

Reply:
Nope, just a typo.

Reply:
Resolved in commit f01a42e